### PR TITLE
add ability to test subset of plugins for integration tests

### PR DESCRIPTION
**Pull request by SaxyPandaBear**
# Description of the issue
See https://github.com/aws/amazon-cloudwatch-agent-test/pull/89

# Description of changes
See https://github.com/aws/amazon-cloudwatch-agent-test/pull/89

This is just for the GitHub Actions YAML side of it to pass the input along to the terraform that is executed from the other repo.

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
See https://github.com/aws/amazon-cloudwatch-agent-test/pull/89

### CODE REVIEW

File: .github/workflows/integrationTest.yml
Comment (sethAmazon) [inline .github/workflows/integrationTest.yml:573]: Will this be empty on merge or throw an error?

**Review by sethAmazon — DISMISSED** [review]

File: .github/workflows/integrationTest.yml
Comment (SaxyPandaBear) [reply .github/workflows/integrationTest.yml:573]: it'll be empty cause the default is `''`

File: .github/workflows/integrationTest.yml
Comment (SaxyPandaBear) [inline .github/workflows/integrationTest.yml:12]: I'll clean this up once I get enough approvals to merge

**Review by williazz — COMMENTED** [review]
Really clean implementation and I look forward to using this feature. I'll approve once the other one goes through and we can point to the aws testing repo

File: .github/workflows/integrationTest.yml
Comment (SaxyPandaBear) [reply .github/workflows/integrationTest.yml:11]: I left it there for testing because things keep getting merged and conflict with my changes. I'll fix it once I have approvals to merge.

**Review by khanhntd — APPROVED** [review]
LGTM. However, it would be more better to add an example for description.

**Review by williazz — APPROVED** [review]
Like Khanh said, I think some documentation about how to use `inputs.plugins` would be helpful

**Pull request merged**
